lib/selenium-webdriver/webdriver.js

```diff
@@ -82,11 +82,33 @@ webdriver.WebElement.equals = function(a, b) {};
 
 /**
  * @return {!webdriver.WebDriver} The parent driver for this instance.
+ *
+ * @view
+ * <ul class="pet">
+ *  <li class="dog">Dog</li>
+ *  <li class="cat">Cat</li>
+ * </ul>
+ *
+ * @example
+ * // Using getDriver to find the parent web element to find the cat li
+ * var liDog = element(by.css('.dog')).getWebElement();
+ * var liCat = liDog.getDriver().findElement(by.css('.cat'));
  */
 webdriver.WebElement.prototype.getDriver = function() {};
 
 
 /**
+ * @view
+ * <ul class="pet">
+ *   <li class="dog">Dog</li>
+ *   <li class="cat">Cat</li>
+ * </ul>
+ *
+ * @example
+ * // returns the dog web element
+ * var dog = element(by.css('.dog')).getWebElement();
+ * expect(dog.getId()).not.toBe(undefined);
```

Comment:
Ok, I understand now.  You should add a line up top explaining what this method actually does, because I assumed it got the `id` attribute of an element